### ISSUE TITLE
Update ReadMe.md for Java 11 requirement

### ISSUE THREAD

**Issue by wwwjames** [hashmapinc/Tempus#1136]
`mvn compile` fails on Java 8, Java 11 is required, at least on `dev` branch.

**Comment by cherrera2001** [comment]
You are correct, this has been updated in the readme and the tempus developer quickstart. Thanks! @wwwjames
